test/FSharpTestProjects/CompileFail/NuGet.Config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
+    <add key="dotnet-core" value="https://www.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
i am doing `test/FSharpTestProjects` like `test\TestProjects`, trying to minimize differences to make it easier to mantain ( if c# works, f# should too, that's the idea )
So i copied the [test/TestProjects/CompileFail/NuGet.Config](https://github.com/dotnet/cli/blob/1cc44de96f9aa506d02f9b4815b66df812a9ef5f/test/TestProjects/CompileFail/NuGet.Config),